<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE strutsX PUBLIC "-//enorth strutsx//EN"
"strutsx.dtd">
<strutsX>
    <other-files>
    <file>strutsx_config/strutsx-wy.xml</file>
    </other-files>

    <default-exception-mapping>/WEB-INF/views/error/def_error.jsp</default-exception-mapping>
    <allow-value-binds>
		<bind type="package">cn.com.enorth.feedback.vo.search</bind>
		<bind type="package">cn.com.enorth.feedback.vo.table</bind>
		<!--<bind type="package">cn.com.enorth.ck.service</bind>-->
		<!--<bind type="package">cn.com.enorth.ck.service.auth</bind>-->
		<!--<bind type="class">cn.com.enorth.ck.service.auth.LoginService</bind>-->
	</allow-value-binds>


    <package name="cms_parent_package" extends="default">
        <upload-config>
            <!-- 如果这里为空，会在当前用户的主目录建立临时目录 -->
            <temp-dir></temp-dir>
            <size-threshold>8000</size-threshold>
            <charset>utf-8</charset>
            <single-file-size-max>1073741824000</single-file-size-max>
            <size-max>1073741820000</size-max>
        </upload-config>
        <global-results>
            <result name="error">/WEB-INF/views/error/def_error.jsp</result>
            <result name="EnorthException">/WEB-INF/views/error/enorth_error.jsp</result>
            <result name="EnorthWarningException">/WEB-INF/views/error/enorth_warning.jsp</result>
            <result name="PrivException">/WEB-INF/views/error/priv_error.jsp</result>
            <result name="AccessException">/WEB-INF/views/error/access_error.jsp</result>
            <result name="SessionValidateException">/WEB-INF/views/error/session_error.jsp</result>
        </global-results>
        
        <global-exception-mappings>
            <exception-mapping result="error" exception="java.lang.Exception" />
            <exception-mapping result="EnorthException" exception="cn.com.enorth.product.exception.EnorthException" />
            <exception-mapping result="EnorthWarningException" exception="cn.com.enorth.product.exception.EnorthWarningException" />
            <exception-mapping result="PrivException" exception="cn.com.enorth.utility.app.filter.priv.impl.common.PrivException" />
            <exception-mapping result="AccessException" exception="cn.com.enorth.utility.app.filter.priv.impl.common.AccessException" />
            <exception-mapping result="SessionValidateException" exception="cn.com.enorth.utility.app.filter.priv.impl.common.SessionValidateException" />
        </global-exception-mappings>
        
    </package>
</strutsX>
